src/pages/sighting/AssetGroupSighting.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
+
+import SightingCore from './SightingCore';
+import useAssetGroupSighting from '../../models/assetGroup/useAssetGroupSighting';
+
+export default function AssetGroupSighting() {
+  const history = useHistory();
+  const { id } = useParams();
+
+  const {
+    data: assetGroupSightingData,
+    loading: assetGroupLoading,
+    error: assetGroupError,
+    statusCode: assetGroupStatusCode,
```

Comment:
I'm not seeing statusCode in the react-query documentation nor in useAssetGroupSighting.js... where is this coming from?

Reply:
Weird it's not in the docs but I'm pretty sure I saw it on the actual object. I can double check this

Reply:
it wasn't there! however statuscode could be found on data.status so i made the adjustment. nice catch